stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.41.1
+version: 1.41.2
```

Comment:
Should increase minor version for new features. `1.42.0`

Reply:
done